### DISCUSSION TITLE
How to get the console log in the report

### DISCUSSION THREAD

**Discussion by mihirsasmal** [jest-community/jest-junit#227] [Q&A]
Is there anyway I can get `<system-out/>` and  `<system-err/>` details in Jest Junit report , similar to mocha junit report.
I need some logs or additional details in the report. How do I do that ?

**Comment by palmerj3** [comment]
https://github.com/jest-community/jest-junit/issues/45

**Reply by mihirsasmal** [reply]
Well, in that case I am not able to use that feature or that feature not working for me.
I have that latest Jest-Junit.
Here is my Jest -Junit config looks like 
```
 ["jest-junit", {
            "suiteName": "jest tests",
            "outputDirectory": ".",
            "outputName": "junit.xml",
            "uniqueOutputName": "false",
            "includeConsoleOutput" : true,
            "includeShortConsoleOutput" : true
          }
        ]
```
Also, I have two test in one file. still I dont get the console output in the report. I have console log in both the test. and the tests are two `describe` and one `it`  inside that.

Can you please suggest me what to do to make it work?

**Comment by palmerj3** [comment]
Jest no longer sends console data to reporters. So jest-junit supports it if you use a very old version of Jest from 2018 or earlier.

But at this time there is nothing I can do to enable this functionality.

**Reply by mihirsasmal** [reply]
Okay :(
Is there anyway I can add any data to the report, that way it will solve the problem.
Like attaching messages works in Jest-html-reporters or addcontext in mochawesome.

Can we do that in Jest-Junit ?

**Comment by palmerj3** [comment]
The only thing jest-junit supports adding content externally to generated junit files is properties. Doubtful it will do what you're trying to do but perhaps. Have a look at the README and look for "testSuitePropertiesFile"
